sequence_processing_pipeline/tests/test_FastQCJob.py

```diff
@@ -18,20 +14,24 @@ def setUp(self):
         self.output_directory = self.path('output_directory')
         self.qiita_job_id = 'abcdabcdabcdabcdabcdabcdabcdabcd'
 
-    def test_creation(self):
+    def test_config_file_not_found(self):
         with self.assertRaises(PipelineError) as e:
             FastQCJob(self.run_dir, self.output_directory, 16, 16,
-                      'sequence_processing_pipeline/tests/bin/not-fastqc',
```

Comment:
Thank you for adding these tests; however, are we missing now one when fastqc doesn't exist or is this being checked somewhere else?